stdlib/obj.ml

```diff
@@ -75,8 +78,10 @@ let extension_constructor x =
     if (tag name) = string_tag then (obj slot : extension_constructor)
     else invalid_arg "Obj.extension_constructor"
 
+[@@inline always]
```

Comment:
Does this `inline` annotation apply to `extension_constructor` or to `extension_name`??